python/kaitai/compress/algorithms/brotli.py

```diff
@@ -0,0 +1,42 @@
+import typing
+from ..core import KaitaiProcessor, ProcessorContextStub
+import brotli
+from enum import Enum
+from functools import partial
+
+class Brotli:
+	__slots__ = ("compressorParams", "decompressorParams", "brotli")
+	def __init__(self, level:typing.Optional[int]=None, mode:typing.Optional[str]="generic", log_window_size:typing.Optional[int]=None, log_block_size:typing.Optional[int]=None, dictionary:typing.Optional[bytes]=None):
+		if self.__class__.dO is None:
+			import brotli
+			self.__class__.brotli = brotli
+	self.compressorParams = {}
+	self.decompressorParams = {}
```

Comment:
I'm not a huge Python expert, but you seem to use tabs here, which is [explicitly prohibited in PEP8](https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces) and allowed only for consistency with large bodies of legacy code.